lib/middleware/websocket/controller.rb

```diff
@@ -33,6 +38,8 @@ def on_close(connection)
     def generate_client(connection:)
       @clients << Client.new(connection: connection)
       @clients.last.generate_player
+
+      Interactor::UpdateClients.new.client_creation(client: @clients.last)
```

Comment:
We're using the same interactor several times here. It might be worth memoizing it to clean things up a bit.
```
def update_clients
  @update_clients ||= Interactor::UpdateClients.new
end
```

Reply:
I'll have to look into what this does, never seen it before, though I know what memoization is

Reply:
@jgayfer fixed.